.travis/install_intel_opencl.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -ev
+
+###########################
+# Get Intel OpenCL Runtime
+###########################
+
+wget http://registrationcenter-download.intel.com/akdlm/irc_nas/12513/opencl_runtime_16.1.2_x64_rh_6.4.0.37.tgz -O /tmp/opencl_runtime.tgz
```

Comment:
Note that the version number or package name may change. Can you put the name and the URL on separate variables ?

Reply:
Is this what you meant?
https://github.com/KhronosGroup/SyclParallelSTL/pull/49/commits/01771ea41018af928df323ba04134858ff820158